src/App.tsx

```diff
@@ -1,12 +1,14 @@
 import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
 import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
 
-
 import LandingPage from './pages/LandingPage'
-import AuthenticatedDashboard from './pages/Dashboard';
+import Dashboard from './pages/Dashboard';
+import DevPage from './pages/DevPage';
+import withAuth from './hocs/WithAuth';
 
 import './App.css'
 
+const AuthenticatedDashboard = withAuth(Dashboard);
```

Comment:
is this double wrapped? Also we might not need the dashboard itself I just added it as a protected component if we needed one/as a proof of concept that the hook was working